carbonmark/components/pages/Home/styles.ts

```diff
@@ -14,6 +14,14 @@ export const global = css`
       display: none;
     }
   }
+
+  display: block;
+  grid-template-columns:
```

Comment:
This could use a short comment on what each of the named values are used for

Reply:
Sure, will do a follow up PR with all the suggestions you have here and will add some comments around this.